shell/platform/fuchsia/dart_runner/mapped_resource.cc

```diff
@@ -97,4 +108,43 @@ MappedResource::~MappedResource() {
   }
 }
 
+bool ElfSnapshot::Load(fdio_ns_t* namespc, const std::string& path) {
+  fuchsia::mem::Buffer vmo;
+  if (!OpenVmo(&vmo, namespc, path, /*executable=*/true))
+    return false;
```

Comment:
Nit: Maybe log here that we could not open the VMO. Would help debug errors.

Reply:
Done.